list_portfolio_fills.go

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2024-present Coinbase Global, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package prime
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/coinbase-samples/core-go"
+)
+
+type ListPortfolioFillsRequest struct {
+	PortfolioId string            `json:"portfolio_id"` // required
+	StartDate   time.Time         `json:"start_date"`   // required
+	EndDate     time.Time         `json:"end_date"`
+	Pagination  *PaginationParams `json:"pagination_params"`
+}
+
+type ListPortfolioFillsResponse struct {
+	Fills      []*OrderFill               `json:"fills"`
```

Comment:
orderfill is missing order_id (order_id is present in addition to id, which is the fill id)

Reply:
Great catch! GetOrderFills was also missing order_id, have added.